### DISCUSSION TITLE
Are GitHub Actions just wonky today?

### DISCUSSION THREAD

**Discussion by nutjob4life** [upptime/upptime#624] [Q&A] [answered]
What is going on with GitHub Actions today (2022-07-27)?

![Screen Shot 2022-07-27 at 11 01 43 AM](https://user-images.githubusercontent.com/814813/181295706-9535302d-9f05-422e-bf00-76ae9eff308f.png)

Two of my Upptime sites are doing this. And apparently it's happening to `upptime/upptime` as well.

The error in the log is always something like:

    Can't use 'tar -xzf' extract archive file: /home/runner/work/_actions/_temp_c13ac3e0-7f66-43bf-8f0c-acf6e22528cc/402c92c0-d577-4285-bd27-03316a4c22ff.tar.gz. return code: 2.

I'm sure this isn't specific to Upptime. Is some data center on fire? 😅

**Comment by leynier** [comment]
It happens to me too

**Comment by http403** [comment, marked as answer]
#623 is tracking this issue. For now it seems there are no solutions yet.
